### DISCUSSION TITLE
Frontend variables in script

### DISCUSSION THREAD

**Discussion by ingfabio** [frangoteam/FUXA#806] [Q&A]
It's possible to access a variable on the internal device (who say frontend only) from a script on the client side?
I want to create a bar chart with different information on each customer. The script makes a query to the database but I can't send it to the chart. If I set the graph to a server variable, all clients see the same thing.

**Comment by unocelli** [comment]
I don't think I understand your issue, why don't you create a different chart and view for each user?
